### PR TITLE
WEBAPP-524: Update readme (again)

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/). 
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **WEBAPP**.

### CODE REVIEW

File: docs/02-technology-stack.md [hunk 1]
```diff
@@ -1,36 +1,49 @@
...
 * [Jest](https://facebook.github.io/jest/) is used for testing.
 * [<img src="https://d2ogrdw2mh0rsl.cloudfront.net/production/images/static/header/header-logo.svg" width="150">](https://www.browserstack.com) is used for testing cross-browser compatibility
-* To test another api, you can set the new cms domain via Console of your web browser: Just type `window.localStorage.setItem('api-url', 'https://cms-test.integreat-app.de')` and reload the page.
 
-### Linting
+### Test CMS
+**Testing with the live cms instance should be avoided.**
+To use the test cms, just set the new cms domain by typing 
+`window.localStorage.setItem('api-url', 'https://cms-test.integreat-app.de')` 
+in the console of your web browser and reloading the page.
```
Comment (Taggotty) [inline docs/02-technology-stack.md:30]:
```suggestion
in the console of your web browser and reload the page.
```

File: docs/02-technology-stack.md [hunk 2]
```diff
@@ -1,36 +1,49 @@
...
 You can run the linter by calling **yarn run lint**. Some issues can be fixed automatically by running **yarn run lint:fix**
 
-### Type checking
+## Type checking
 [Flow](https://flow.org/) is used for static type checking.
 
 You can run flow using **yarn flow**. 
+
+## Backend
+The endpoints for the webapp and this project are defined in the [api-client](https://github.com/Integreat/api-client).
```
Comment (Taggotty) [inline docs/02-technology-stack.md:45]: link doesn't exist

**Comment by Taggotty** [comment]
maybe you could also take a look at https://github.com/Integreat/integreat-webapp/tree/master/tools , as e.g. findMissingLocales.js seems to have moved to the locales project.

File: docs/02-technology-stack.md (hunk 1 above)
Comment (klinzo) [reply docs/02-technology-stack.md:30]: I think this is fine :D Although it is probably not the easiest sentence.

File: docs/02-technology-stack.md (hunk 2 above)
Comment (klinzo) [reply docs/02-technology-stack.md:45]: Good catch!

**Pull request merged**